src/app.js

```diff
@@ -0,0 +1,84 @@
+// Debemos poder leer un archivo.
+// Asegurarnos que sea un archivo Markdown.
+// Obtener los links de este archivo
+// Validar esos links, que me devuelva un status 200
+
+const {validateLinks, linkStats, validateStats } = require('./main.js');
+
+const fs = require('fs');
+const marked = require('marked');
+const path = require('path');
+//const colors = require('colors');
+
+let index = process.argv.indexOf("--file");
+let flags = process.argv;
+
+function readFile(uri) { 
+    if (fs.existsSync(uri)){
+        let fileContent = fs.readFileSync(uri, 'utf8');
+        return fileContent;
+    } else {
+        console.log('El archivo'.red, uri.red.bgCyan, 'NO existe'.red);
+        return false;
+    };
+};
```

Comment:
Definir mejor el parametro recibido 
```suggestion
function readFile(path) { 
    if (fs.existsSync(path)){
        let fileContent = fs.readFileSync(path, 'utf8');
        return fileContent;
    } else {
        console.log('El archivo'.red, uri.red.bgCyan, 'NO existe'.red);
        return false;
    };
};
```